framework/src/governance/lib/resources/datazone-msk-authorizer-grant/index.mjs

```diff
@@ -0,0 +1,88 @@
+import { KafkaClient, PutClusterPolicyCommand } from "@aws-sdk/client-kafka"
+import { IAMClient, PutRolePolicyCommand } from "@aws-sdk/client-iam";
+
+
+
+function getIamResources(region, account, clusterName, clusterUuid, topic) {
+  return [
+    `arn:aws:kafka:${region}:${account}:topic/${clusterName}/${clusterUuid}/${topic}`,
+    `arn:aws:kafka:${region}:${account}:cluster/${clusterName}/${clusterUuid}`,
+    `arn:aws:kafka:${region}:${account}:group/${clusterName}/${clusterUuid}/*`
+  ]
+}
+
+const iamActions = [
```

Comment:
Lets rename this to the `readAction` for readability and comprehension.

Reply:
done